tests/foreman/api/test_multiple_paths.py

```diff
@@ -531,9 +495,9 @@ class and name and resource_type fields are populated
 
         :CaseImportance: Critical
         """
-        perm = entities.Permission().search(query={'per_page': 1})[0]
-        self.assertGreater(len(perm.name), 0)
-        self.assertGreater(len(perm.resource_type), 0)
+        perm = entities.Permission().search(query={'per_page': '1'})[0]
+        assert len(perm.name) > 0
+        assert len(perm.resource_type) > 0
```

Comment:
non-blocking, but cleaner imo
```suggestion
        assert perm.name
        assert perm.resource_type
```

Reply:
Done.